airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -354,8 +354,19 @@ def read_pod(self, pod: V1Pod) -> V1Pod:
         except BaseHTTPError as e:
             raise AirflowException(f'There was an error reading the kubernetes API: {e}')
 
+    def await_xcom_sidecar_container_start(self, pod: V1Pod) -> None:
+        while True:
+            self.log.info("Checking if xcom sidecar container is started.")
+            if self.container_is_running(pod, PodDefaults.SIDECAR_CONTAINER_NAME):
+                self.log.warning("The xcom sidecar container is started.")
+                break
+            self.log.warning("The xcom sidecar container is not yet started.")
```

Comment:
thinking maybe this we should only log _once_ -- the first time it has to wait, and not every loop.  wdyt?